compiler/noirc_frontend/src/parser/parser.rs

```diff
@@ -1452,11 +1270,6 @@ mod test {
         );
     }
 
-    #[test]
```

Comment:
why was this test removed?

Reply:
Because we had it in `parser/types.rs` as well here: https://github.com/noir-lang/noir/pull/5229/files/cbe9e2fb178047e7c23060569ab845f96ddaa98c#diff-c8f6c64d7457320133a6d819dfb7aca1ac1b70b2e6bfe06e0979afd38838723bR243